runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -349,12 +443,14 @@ public void removeDestination(String logicalGroup, List<String> remoteDestinatio
     }
 
     /**
-     * Get the destinations associated with a specific logical group.
+     * Gets the destinations associated with a specific logical group.
      *
      * @param logicalGroup The group whose destinations will be returned.
-     * @return The associated destinations as a list of strings.
+     *                     This parameter is case-sensitive.
+     * @return The associated destinations as a list of strings, null if logical group is not found.
+     * @throws IllegalArgumentException If logicalGroup is null or empty.
      */
-    public List<String> getDestinations(String logicalGroup) {
+    public List<String> getDestination(String logicalGroup) {
```

Comment:
any reason it was changed destinations -> destination?

Reply:
Plural seems more readable since we are returning a list right?

Reply:
The motive behind changing to everything to singular was to simplify the method signatures so that the caller does not have to discern between singular/plural. Would we want to migrate all methods to plural? In that case addDestinations (and remove) could be used for a single destination as well, which is fine just something to note.